tests/console/snapper_cleanup.pm

```diff
@@ -136,7 +136,10 @@ sub run {
     # tidy up and restore default settings
     assert_script_run("snapper set-config NUMBER_LIMIT=0; snapper cleanup number; rm -fv data", 300);
     assert_script_run("snapper set-config NUMBER_LIMIT=$number_limit_pre NUMBER_MIN_AGE=$number_min_age_pre");
-    assert_script_run("snapper get-config; snapper ls");    # final report
+    assert_script_run("snapper get-config");    # final report
+        # command 'snapper ls' runs timeout or it stucks because of display used space, so we have to disable it on tumbleweed, see poo#122557
+    my $command_args = (is_tumbleweed && check_var('MACHINE', '64bit')) ? '--disable-used-space' : undef;
```

Comment:
Is it too strict with  check_var('MACHINE', '64bit')? 
We may use 64bit-2G as a machine type, then we may hit a new issue :)

Reply:
well, I want just limit this change to tw 64bit, I checked that tw has at moment only this test run with 64 bit qemuram 1.5. I don't care about the future :) if it has 2G then it migt be okay anyway... the strange thing it that this test module passed everywhere except on tw with very limited allocation of cpu or mem.

Reply:
Do you mind checking the qemuram as well?